File: 2017-10-04 Més Python/p1.py
from jutge import read

# La funció maxim, donats dos enters retorna un enter que és el seu màxim.
def maxim(a, b):
    if a > b:
        return a
    else:
        return b
    
# Programa que llegeix dos enters i n'escriu el seu màxim, cridant maxim().    
x, y = read(int, int)
print(maxim(x, y))

File: 2017-09-22 Funcionem!/p1.cc
#include <iostream>
using namespace std;


// La funció maxim, donats dos enters retorna un enter que és el seu màxim.
int maxim(int a, int b) {
    if (a > b) {
        return a;
    } else {
        return b;
    }
}


// Programa que llegeix dos enters i n'escriu el seu màxim, cridant maxim().
int main() {
    int x, y;
    cin >> x >> y;
    cout << maxim(x, y) << endl;
}